crates/epaint/benches/benchmark.rs

```diff
@@ -223,6 +223,36 @@ fn thin_large_line_uv(c: &mut Criterion) {
     });
 }
 
+fn from_rgba_unmultiplied_0(c: &mut Criterion) {
+    c.bench_function("from_rgba_unmultiplied_0", move |b| {
+        let [r, g, bl, a] = black_box([20u8, 20, 20, 0]);
+        b.iter(|| {
+            let color = ecolor::Color32::from_rgba_unmultiplied(r, g, bl, a);
+            black_box(color);
+        });
+    });
+}
+
+fn from_rgba_unmultiplied_100(c: &mut Criterion) {
+    c.bench_function("from_rgba_unmultiplied_100", move |b| {
+        let [r, g, bl, a] = black_box([20u8, 20, 20, 100]);
```

Comment:
Thanks for adding a proper benchmark for this! I think a more fair benchmark would test a bunch of _different_ color values though. Now it will hit the same place in the LUT every time, so it will always be a cache hit.